hack/builder/Dockerfile

```diff
@@ -92,6 +92,12 @@ RUN set -x && \
 
 RUN pip3 install j2cli operator-courier==1.3.0
 
+RUN set -x && \
+    wget https://github.com/vmware-tanzu/sonobuoy/releases/download/v0.19.0/sonobuoy_0.19.0_linux_amd64.tar.gz && \
+    tar xvf sonobuoy_0.19.0_linux_amd64.tar.gz && \
```

Comment:
on kubevirtci, it always take the latest version of sonobuoy isnt it ?
i think we should do the same and not have a fixed version ?

Reply:
All the other dependencies here are pinned to a specific version. The builder image is built and published manually, so I assume we want to have close control over our dependencies here.

Also, we don't run Kubernetes conformance tests. We only use sonobuoy to run our own conformance plugin.